les7.js

```diff
@@ -0,0 +1,9 @@
+const a = [1, 2, 3, "1", "2"];
+const newArray = a.map( x => {
+    if (typeof x === "number") {
+        return String(x);
+    } else if (typeof x === "string") {
```

Comment:
you know that entire array contains only 2 value type. that's why you can use only one `if statement`